src/components/pill/pill.style.js

```diff
@@ -113,6 +115,18 @@ const PillStyle = styled.span`
       `}
 
       ${isClassic(theme, colorVariant) && classicThemeForPill(colorVariant, inFill, isDeletable)}
+
+      ${theme.name === 'medium' && css`
+        font-size: 12px;
+        border-radius: 16px;
+        padding-bottom: 4px;
+      `}
+
+      ${theme.name === 'large' && css`
+        font-size: 14px;
+        border-radius: 24px;
+        padding-bottom: 4px;
+      `}
```

Comment:
Are you sure that font sizes for small business, medium business and large business themes should differ? I think that the size should be a separate prop.

Reply:
^^^ agreed

Reply:
agree, size is now separate property.